angular/core/components/proposal_form/proposal_form.coffee

```diff
@@ -14,6 +14,9 @@ angular.module('loomioApp').factory 'ProposalForm', ->
       successEvent: 'proposalCreated'
       successCallback: ->
         $rootScope.$broadcast 'setSelectedProposal'
+        Records.attachments.find(attachableId: proposal.id, attachableType: 'Proposal')
```

Comment:
I think this should be `attachableType: 'Motion'`, since that's how it's stored in the DB.